src/alire/alire-properties-labeled.adb

```diff
@@ -132,4 +131,42 @@ package body Alire.Properties.Labeled is
       return Conditional.Properties
       renames From_TOML_Project_File_Cases_Internal;
 
+   --------------
+   -- Validate --
+   --------------
+
+   procedure Validate (L    : Label;
+                       From : TOML_Adapters.Key_Queue) is
+   begin
+      case L.Name is
+         when Description =>
+            if L.Value'Length > Max_Description_Length then
+               From.Checked_Error ("Description string is too long");
+            end if;
+
+         when Maintainer =>
+            if not Utils.Could_Be_An_Email (L.Value, With_Name => True) then
+               From.Checked_Error
+                 ("Maintainers must have a valid email, but got: "
+                  & L.Value);
+            end if;
+
+         when Maintainers_Logins =>
+            if not Utils.Could_Be_An_Email (L.Value, With_Name => False) then
```

Comment:
GitHub logins are not emails in this case. For instance mine is `Fabien-Chouteau`, yours is `mosteo`. This is what my check-authors action will look for.

Reply:
Oooops. Right.